src/firebase/CRUDServices/user.js

```diff
@@ -0,0 +1,114 @@
+export default class User {
+    /**
+     * Private fields for user object.
+     */
+    #id;
+    #username;
+    #password;
+    #firstName;
+    #lastName;
+    #email;
+    #role;
```

Comment:
What does the #variable mean?

Reply:
@quanganhhoang it appears `#` is the proposed JS syntax for private fields in a Class. However, this is not standard yet and according to the [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields) it's not widely supported.

Reply:
I thought I have to use this style since this was out more than a year ago. Didn't think of that react might not support this. I can revise this to the preferred way.